examples/single_agent.py

```diff
@@ -78,14 +67,24 @@ def main(scenarios, sim_name, headless, num_episodes, seed, max_episode_steps=No
     env.close()
 
 
+def _build_scenario(scenario):
+    build_scenario = f"scl scenario build-all --clean {scenario}"
+    os.system(build_scenario)
```

Comment:
`subprocess.Popen()` is more flexible than `os.system` and i suggest we should use that instead.

Reply:
Changed.